🐛 [BUG] Crash Empty Module Name

### Description

I don't even know what happened

### Logs

```bash
[tree_node-1] Traceback (most recent call last):
[tree_node-1]   File "/home/ubuntu/ws/install/ros_bt_py/lib/ros_bt_py/tree_node", line 33, in <module>
[tree_node-1]     sys.exit(load_entry_point('ros-bt-py', 'console_scripts', 'tree_node')())
[tree_node-1]              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[tree_node-1]   File "/home/ubuntu/ws/build/ros_bt_py/ros_bt_py/tree_node.py", line 508, in main
[tree_node-1]     executor.spin()
[tree_node-1]   File "/opt/ros/kilted/lib/python3.12/site-packages/rclpy/executors.py", line 374, in spin
[tree_node-1]     self.spin_once()
[tree_node-1]   File "/opt/ros/kilted/lib/python3.12/site-packages/rclpy/executors.py", line 1047, in spin_once
[tree_node-1]     self._spin_once_impl(timeout_sec)
[tree_node-1]   File "/opt/ros/kilted/lib/python3.12/site-packages/rclpy/executors.py", line 1044, in _spin_once_impl
[tree_node-1]     future.result()  # raise any exceptions
[tree_node-1]     ^^^^^^^^^^^^^^^
[tree_node-1]   File "/opt/ros/kilted/lib/python3.12/site-packages/rclpy/task.py", line 116, in result
[tree_node-1]     raise exception
[tree_node-1]   File "/opt/ros/kilted/lib/python3.12/site-packages/rclpy/task.py", line 321, in _execute_coroutine_step
[tree_node-1]     result = coro.send(None)
[tree_node-1]              ^^^^^^^^^^^^^^^
[tree_node-1]   File "/opt/ros/kilted/lib/python3.12/site-packages/rclpy/executors.py", line 630, in handler
[tree_node-1]     await call_coroutine()
[tree_node-1]   File "/opt/ros/kilted/lib/python3.12/site-packages/rclpy/executors.py", line 564, in _execute
[tree_node-1]     response = await await_or_execute(srv.callback, request, srv.srv_type.Response())
[tree_node-1]                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[tree_node-1]   File "/opt/ros/kilted/lib/python3.12/site-packages/rclpy/executors.py", line 138, in await_or_execute
[tree_node-1]     return callback(*args)
[tree_node-1]            ^^^^^^^^^^^^^^^
[tree_node-1]   File "/home/ubuntu/ws/build/ros_bt_py/ros_bt_py/tree_manager.py", line 143, in service_handler
[tree_node-1]     return func(self, request, response, **kwds)
[tree_node-1]            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[tree_node-1]   File "/home/ubuntu/ws/build/ros_bt_py/ros_bt_py/tree_manager.py", line 1997, in set_options
[tree_node-1]     new_node = node.__class__(
[tree_node-1]                ^^^^^^^^^^^^^^^
[tree_node-1]   File "/home/ubuntu/ws/build/ros_bt_py/ros_bt_py/ros_nodes/topic.py", line 404, in __init__
[tree_node-1]     self._topic_type = self.options["topic_type"].get_type_obj()
[tree_node-1]                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[tree_node-1]   File "/home/ubuntu/ws/build/ros_bt_py/ros_bt_py/custom_types.py", line 83, in get_type_obj
[tree_node-1]     return rosidl_runtime_py.utilities.get_message(self.type_str)
[tree_node-1]            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[tree_node-1]   File "/opt/ros/kilted/lib/python3.12/site-packages/rosidl_runtime_py/utilities.py", line 28, in get_message
[tree_node-1]     interface = import_message_from_namespaced_type(get_message_namespaced_type(identifier))
[tree_node-1]                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[tree_node-1]   File "/opt/ros/kilted/lib/python3.12/site-packages/rosidl_runtime_py/import_message.py", line 30, in import_message_from_namespaced_type
[tree_node-1]     module = importlib.import_module(
[tree_node-1]              ^^^^^^^^^^^^^^^^^^^^^^^^
[tree_node-1]   File "/usr/lib/python3.12/importlib/__init__.py", line 90, in import_module
[tree_node-1]     return _bootstrap._gcd_import(name[level:], package, level)
[tree_node-1]            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[tree_node-1]   File "<frozen importlib._bootstrap>", line 1384, in _gcd_import
[tree_node-1]   File "<frozen importlib._bootstrap>", line 1298, in _sanity_check
[tree_node-1] ValueError: Empty module name
[ERROR] [tree_node-1]: process has died [pid 14173, exit code 1, cmd '/home/ubuntu/ws/install/ros_bt_py/lib/ros_bt_py/tree_node --ros-args -r __ns:=/ --params-file /tmp/launch_params_u21xtshy --params-file /tmp/launch_params_tc8wagsj --params-file /tmp/launch_params_uwwn94w6 --params-file /tmp/launch_params_tb5_6ixs --params-file /tmp/launch_params_w7t3apkz --params-file /tmp/launch_params_3nanfseb --params-file /tmp/launch_params_9oos4kgd --params-file /tmp/launch_params_amevjdv4 --params-file /tmp/launch_params_0khq_7ax'].
```

### ROS 2 Version

_No response_

### OS

_No response_

--- a/ros_bt_py/ros_bt_py/data_types.py
+++ b/ros_bt_py/ros_bt_py/data_types.py
@@ -1314,7 +1314,7 @@ class RosMessageType(RosContainer):
                     config = d
             config["message_type"] = message_type
             return Ok(config)
-        except AttributeError:
+        except (AttributeError, ImportError, ValueError):
             return Err(f"Cannot find message type '{msg.ros_msg_type}'")
 
     def is_compatible(

--- a/ros_bt_py/tests/test_data_types.py
+++ b/ros_bt_py/tests/test_data_types.py
@@ -1,0 +1,41 @@
+# Copyright (c) 2026 FZI Forschungszentrum Informatik
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+#
+#    * Redistributions of source code must retain the above copyright
+#      notice, this list of conditions and the following disclaimer.
+#
+#    * Redistributions in binary form must reproduce the above copyright
+#      notice, this list of conditions and the following disclaimer in the
+#      documentation and/or other materials provided with the distribution.
+#
+#    * Neither the name of the copyright holder nor the names of its
+#      contributors may be used to endorse or promote products derived from
+#      this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+# LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+# CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+# SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+# CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+# ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
+
+from ros_bt_py.data_types import get_iotype_for_msg
+from ros_bt_py_interfaces.msg import NodeDataType
+
+
+def test_invalid_ros_message_type_returns_error():
+    result = get_iotype_for_msg(
+        NodeDataType(
+            type_identifier=NodeDataType.ROS_INTERFACE_VALUE,
+            ros_msg_type="",
+        )
+    )
+
+    assert result.is_err()

